src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -34,7 +34,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(CallbackValidations.class);
 
-    private static final DateTimeFormatter FORMATTER = new DateTimeFormatterBuilder()
+    // todo review usage
+    public static final DateTimeFormatter FORMATTER = new DateTimeFormatterBuilder()
```

Comment:
Not necessarily here, but because we're using this formatter not just from CallbackValidations, it's best if it goes to a more common place.

Reply:
exactly. i'm trying to fix as quickly as possible. to get the happy path working. together with @apereverzin functional tests. looks like this one is last conceptually. now we are hitting the wall while respond gets back from a third party dependency :/